notification-manager/README.md

```diff
@@ -1,7 +1,8 @@
 # Notification manager
-Service for sending emails about payments
+### Service for sending emails about payments.
 
-### Start service
-- Open package resources
-- Write username (your email) and password in application.yaml
-- start service
\ No newline at end of file
+## How to run
+- Open file ***src/main/resources/application.yaml***.
+- Write ***username*** (your email) and ***password*** in application.yaml.
```

Comment:
I assume you change your gmail profile a bit to allow 3rd party apps communicate with your email.
If yes, add on the next line that they have to setup their host email as well